calico/datamodel_v1.py

```diff
@@ -44,6 +44,9 @@
 POLICY_DIR = VERSION_DIR + '/policy'
 PROFILE_DIR = POLICY_DIR + "/profile"
 
+# Key used for leader election by Neutron mechanism drivers.
+NEUTRON_ELECTION_KEY = VERSION_DIR + '/neutron_election'
```

Comment:
Agree, and agree. I'll move it for now but leave it lower-case, I'll let you come up with some naming conventions! =D